analyzer/src/funTest/assets/projects/synthetic/dotnet-direct-dependencies-only-expected-output.yml

```diff
@@ -168,9 +259,29 @@ packages:
     url: ""
     revision: ""
     path: ""
-issues:
-- timestamp: "1970-01-01T00:00:00Z"
-  source: "NuGet"
-  message: "Failed to get package data for 'NuGet::foobar:1.2.3' from any of [https://api.nuget.org/v3/registration5-gz-semver2,\
-    \ https://pkgs.dev.azure.com/ms/20953fb4-d427-4ad6-b78c-13142d0c5462/_packaging/64e3bf16-3e04-41cd-b90f-60fdc9ccf494/nuget/v3/registrations2-semver2]."
-  severity: "ERROR"
+- id: "NuGet::foobar:1.2.3"
```

Comment:
@sschuberth One cannot assume that a package is "dummy" just because it does not exist in a NuGet repository. It could be a local non-published package. Note that this is the same behavior that we have in the python-inspector: we return these packages but they do not have any public URLs and other related attributes retrieved from repository.

Reply:
> One cannot assume that a package is "dummy" just because it does not exist in a NuGet repository. It could be a local non-published package.

Ok, I agree now that the originally expected error should not be there (at least not with that particular wording of the error message). But IMO the "foobar" package should not be there, either, because if we have no metadata whatsoever, doesn't that mean that the package is also not available locally? Or phrased the other way around: Would `nuget-inspector` report metadata (like at least the VCS location) for local packages?

If yes, when a package is empty except for the name and version, it's probably a safe indicator to say that the package does not exist, and that we should not include it, but record an error.

Reply:
> Or phrased the other way around: Would nuget-inspector report metadata (like at least the VCS location) for local packages?

If a local package has metadata, they would be picked. It it common for local packages to be low on metadata though.